cilium-health/launch/endpoint.go

```diff
@@ -126,7 +126,7 @@ func configureHealthInterface(netNS ns.NetNS, ifName string, ip4Addr, ip6Addr *n
 			name := fmt.Sprintf("net.ipv6.conf.%s.disable_ipv6", ifName)
 			// Ignore the error; if IPv6 is completely disabled
 			// then it's okay if we can't write the sysctl.
-			_ = sysctl.Write(name, "1")
+			_ = sysctl.Enable(name)
```

Comment:
FWIW the reason this wasn't `Enable()` to begin with is that it's enabling the option to disable IPv6, so the casual reader might scan through and think "IPv6.... enable" but really this is the reverse >.<

Reply:
Makes sense, change reverted.